source/uwp/Renderer/AdaptiveCardRenderer.vcxproj

```diff
@@ -194,6 +195,7 @@
     <ClInclude Include="lib\Vector.h" />
     <ClInclude Include="lib\WholeItemsPanel.h" />
     <ClInclude Include="lib\XamlBuilder.h" />
+    <ClCompile Include="..\..\shared\cpp\ObjectModel\MarkDownParser.cpp" />
```

Comment:
>..\..\shared\cpp\ObjectModel\MarkDownParser.cpp [](start = 24, length = 47)

You shouldn't need the changes to this file.

Reply:
yes, you are correct.

---
In reply to: [158146430](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158146430) [](ancestors = 158146430)

Reply:
i will remove those before check-in

---
In reply to: [158147945](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158147945) [](ancestors = 158147945,158146430)

Reply:
removed

---
In reply to: [158162804](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158162804) [](ancestors = 158162804,158147945,158146430)